docs/adrs/00005-license-export-analytics.md

```diff
@@ -0,0 +1,332 @@
+   "properties": [
+      ...
+   ]
+},
+```
+Field text from license of licenses node,spdx_licenses from parsed by license node.
+
+### Current license data structure design in trustify
+**I use an example to demonstrate where the existing data structure does not meet the requirements.**
+```rust
+#[derive(Debug, FromQueryResult)]
+pub struct SbomLicenseBase {
+   pub sbom_name: Option<String>,
+   pub sbom_namespace: Option<String>,
+   pub component_group: Option<String>,
+   pub component_version: Option<String>,
+   pub node_id: String,
+   pub sbom_id: Uuid,
+   pub package_name: String,
+   pub version: String,
+   pub purl: serde_json::Value,
+   pub cpe:  Option<Vec<String>>,
+   pub text: Option<String>,
+   pub spdx_licenses: Option<Vec<String>>,
+   pub spdx_license_exceptions: Option<Vec<String>>,
+}
+
+#[test_context(TrustifyContext)]
+#[test(tokio::test)]
+async fn test_spdx(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
+   let _result = ctx
+           .ingest_document("quarkus-bom-2.13.8.Final-redhat-00004.json")
+           .await?;
+   let sbom_license_base: Vec<SbomLicenseBase> = sbom::Entity::find()
+           .join(JoinType::LeftJoin, sbom::Relation::Packages.def())
+           .join(JoinType::Join, sbom_package::Relation::Node.def())
+           .join(JoinType::LeftJoin, sbom_package::Relation::Purl.def())
+           .join(JoinType::Join, sbom_package_purl_ref::Relation::Purl.def())
+           .join(
+              JoinType::LeftJoin,
+              qualified_purl::Relation::VersionedPurl.def(),
+           )
+           .join(
+              JoinType::LeftJoin,
+              versioned_purl::Relation::LicenseAssertions.def(),
+           )
+           .join(
+              JoinType::LeftJoin,
+              purl_license_assertion::Relation::License.def(),
+           )
+           .select_only()
+           .column_as(sbom::Column::SbomId, "sbom_id")
+           .column_as(sbom_package::Column::NodeId, "node_id")
+           .column_as(sbom_package::Column::Version, "version")
+           .column_as(sbom_node::Column::Name, "package_name")
+           .column_as(qualified_purl::Column::Purl, "purl")
+           .column_as(license::Column::Text, "text")
+           .column_as(license::Column::SpdxLicenses, "spdx_licenses")
+           .column_as(
+              license::Column::SpdxLicenseExceptions,
+              "spdx_license_exceptions",
+           )
+           .into_model::<SbomLicenseBase>()
+           .all(&ctx.db)
+           .await?;
+  
+   Ok(())
+}
+```
+As shown in the above example, when the SPDX SBOM was created, the sbom_name and sbom_namespace could not be obtained. Additionally, we are currently storing both licenseDeclared and licenseConcluded, and we are unable to distinguish between them.
+At same time, if the SBOM is cycloneDX, sbom_name, component_group, component_version will miss.
+
+**The information in the ExtractedLicensingInfos section is completely missing.**
+## Decision
```

Comment:
I still struggle with the section. As there seem to be issues up until this point, I am not sure it makes sense coming to a conclusion at this point.